lib/bolt_server/transport_app.rb

```diff
@@ -286,12 +295,23 @@ def make_ssh_target(target_hash)
     end
 
     def make_winrm_target(target_hash)
-      overrides = {
-        'protocol' => 'winrm'
+      defaults = {
+        'ssl' => false,
+        'ssl-verify' => false
+      }
+
+      opts = defaults.merge(target_hash.clone)
```

Comment:
Why do you have to clone the target_hash?

Reply:
I just copied what was in `make_ssh_target` already.

Reply:
I think that one is cloned bc `overrides` is merged over it.

Reply:
But i dont think it is needed there either.

Reply:
Removed `clone`.